src/main/kotlin/no/nav/familie/ba/mottak/task/VurderLivshendelseTask.kt

```diff
@@ -92,53 +94,54 @@ class VurderLivshendelseTask(
         }
     }
 
-    private fun finnRelatertePersonerMedSak(
+    private fun finnBrukereMedLøpendeSakInvolverendePerson(
             personIdent: String,
             pdlPersonData: PdlPersonData
     ): Set<RestFagsakDeltager> {
 
-        val personerMedSak = mutableSetOf<RestFagsakDeltager>()
-
-        val familierelasjon = pdlPersonData.forelderBarnRelasjon
-        // populerer en liste med barn for person. Hvis person har barn, så sjekker man etter løpende sak
-        val listeMedBarn =
-                familierelasjon.filter { it.minRolleForPerson != FORELDERBARNRELASJONROLLE.BARN }
-                        .map { it.relatertPersonsIdent }
-        secureLog.info("finnRelatertePersonerMedSak(): listeMedBarn size = ${listeMedBarn.size} identer = " +
-                       "${listeMedBarn.fold("") { identer, it -> identer + " " + it }}")
-        if (listeMedBarn.isNotEmpty()) {
-            personerMedSak += sakClient.hentRestFagsakDeltagerListe(personIdent).filter {
-                secureLog.info("finnRelatertePersonerMedSak(): Hentet Fagsak for person ${personIdent}: ${it.fagsakId} ${it.fagsakStatus}")
+        val brukereMedLøpendeSakInvolverendePerson = mutableSetOf<RestFagsakDeltager>()
+
+        val familierelasjoner = pdlPersonData.forelderBarnRelasjon
+
+        // Hvis person har barn, så sjekker man etter løpende sak på person
+        val personHarBarn = familierelasjoner.filter { it.erBarn }.let {
+                secureLog.info("finnBrukereMedLøpendeSakInvolverendePerson(): listeMedBarn size = ${it.size} identer = " +
+                                       it.fold("") { identer, it -> identer + " " + it.relatertPersonsIdent })
+                it.isNotEmpty()
+            }
+
+        if (personHarBarn) {
+            brukereMedLøpendeSakInvolverendePerson += sakClient.hentRestFagsakDeltagerListe(personIdent).filter {
+                secureLog.info("finnBrukereMedLøpendeSakInvolverendePerson(): Hentet Fagsak for person ${personIdent}: ${it.fagsakId} ${it.fagsakStatus}")
                 it.fagsakStatus != AVSLUTTET
             }
         }
 
-        secureLog.info("finnRelatertePersonerMedSak(): personerMedSak.size = ${personerMedSak.size}")
+        secureLog.info("finnBrukereMedLøpendeSakInvolverendePerson(): personerMedSak.size = ${brukereMedLøpendeSakInvolverendePerson.size}")
 
-        // Sjekker om foreldrene til person under 19 har en løpende sak.
-        if (pdlPersonData.fødsel.isEmpty() ||
```

Comment:
Alltid oppfylt, siden denne informasjonen ikke er del av noen av graphql-query'ene som brukes

Reply:
Er mulig å utvide graphql da

Reply:
Det vil jo begrense kall mot ba-sak. Er vel ikke nødvendig å sjekke om man har løpende barnetrygd for en person hvor barnet er 86 og forelder er 110.

Reply:
Sant nok. Kan fikse og legge tilbake den. Men bør fortsatt ha med barnet i kallet mot ba-sak, tenker jeg